src/renderer.js

```diff
@@ -6,41 +7,54 @@ function getMapping() {
     ]
 }
 function init() {
-    window.config = {};
-    window.config = window.electron.getConfig();
     addListeners();
     populateFields();
 }
 
 async function populateFields() {
-    let config = window.config;
-    let mapping = getMapping();
-    for (map of mapping) {
-        let input = document.getElementById(map.id);
-        input.setAttribute("value", config[map.key])
-    }
+    config = window.electron.getConfig();
+    let profile = document.getElementById("profile");
```

Comment:
Np: you may want to make id as constants for reuse. See another usage at line 27